docs/contracts/functions.rst

```diff
@@ -229,7 +231,8 @@ This behaviour is also in line with the ``STATICCALL`` opcode.
 Receive Ether Function
 ======================
 
-A contract can have at most one ``receive`` function, declared using ``receive() external payable``.
+A contract can have at most one ``receive`` function, declared using ``receive() external payable``
+(without the ``function`` keyword).
```

Comment:
Adding this clarifies in ways, but now I'm confused in other ways, what does the syntax look like? Perhaps an example is useful.